docker/Dockerfile.firedrake

```diff
@@ -0,0 +1,48 @@
+FROM firedrakeproject/firedrake-vanilla:2024-02
+
+MAINTAINER Joe Wallwork <joe.wallwork@outlook.com>
```

Comment:
A warning is raised that `MAINTAINER` is deprecated and that `LABEL` should be used (https://docs.docker.com/reference/dockerfile/#maintainer-deprecated)

Reply:
See #23.